### ISSUE TITLE
Android paymentRequestWithCardForm how to change text of cancel button

### ISSUE THREAD

**Issue by denis-shostik** [tipsi/tipsi-stripe#288]
on Android devices text for canceling has language which relate to current system language, I needn't that one. How to pass custom there text, pls look screenshot https://prnt.sc/jgg3xy

**Comment by isnifer** [comment]
Hi @denis-shostik, currently these strings defined [here](https://github.com/tipsi/tipsi-stripe/blob/master/android/src/main/res/values/strings.xml).
A line in the code that interested for you is [here](https://github.com/tipsi/tipsi-stripe/blob/master/android/src/main/java/com/gettipsi/stripe/dialog/AddCardDialogFragment.java#L88).
So, you can create a PR that will bring `Cancel` word `strings`:
```xml
<string name="gettipsi_card_enter_dialog_negative_button">Cancel</string>
```

And get it in `AddCardDialogFragment.java`:
```diff
- .setNegativeButton(android.R.string.cancel, null).create();
+ .setNegativeButton(R.string.gettipsi_card_enter_dialog_negative_button, null).create();
```

**Comment by isnifer** [comment]
Fixed in #325

**Comment by isnifer** [comment]
@denis-shostik you're welcome.
